.github/ISSUE_TEMPLATE/mod_crash.yml

```
@@ -1,21 +1,22 @@
-name: Mod Incompatibility
-description: Report an incompatible mod
-labels: [compat]
+name: Mod Crash
+description: Report mod(pack)s that crashes the game.
+labels: [compat, crash]
 body:
   - type: markdown
     attributes:
       value: "## Welcome!"
   - type: markdown
     attributes:
       value: |
-        Thank you for taking your time to report this incompatibility!
-        Make sure you are running the latest version of Connector and its dependencies before reporting.
+        Thank you for taking your time to report this incompatibility! You gotta make sure these before reporting...
+        - You are running the latest version of Connector and its dependencies.
+        - This bug wasn't already reported or being closed.
```

Wording can be improved here
```suggestion
        - This bug wasn't already reported or closed as not planned.
```